python/adjoint/__init__.py

```diff
@@ -15,3 +15,5 @@
 from .filter_source import FilteredSource
 
 from .filters import *
+
+from . import jax
```

Comment:
Does this mean that `meep.adjoint` will break for anyone who doesn't have JAX installed?

Reply:
Yes. I've added a try / except around this for now?